common/web/keyboard-processor/tests/dom/web-test-runner.config.mjs

```diff
@@ -0,0 +1,62 @@
+// @ts-check
+import { devices, playwrightLauncher } from '@web/test-runner-playwright';
+import { esbuildPlugin } from '@web/dev-server-esbuild';
+import { defaultReporter, summaryReporter } from '@web/test-runner';
+import { LauncherWrapper, sessionStabilityReporter } from '@keymanapp/common-test-resources/test-runner-stability-reporter.mjs';
+import { importMapsPlugin } from '@web/dev-server-import-maps';
+import { dirname, resolve } from 'path';
+import { fileURLToPath } from 'url';
+
+const dir = dirname(fileURLToPath(import.meta.url));
+const KEYMAN_ROOT = resolve(dir, '../../../../..');
+
+/** @type {import('@web/test-runner').TestRunnerConfig} */
+export default {
+  // debug: true,
+  browsers: [
+    new LauncherWrapper(playwrightLauncher({ product: 'chromium' })),
+    new LauncherWrapper(playwrightLauncher({ product: 'firefox' })),
+    new LauncherWrapper(playwrightLauncher({ product: 'webkit', concurrency: 1 })),
+  ],
+  concurrency: 10,
+  nodeResolve: true,
+  files: [
+    '**/*.spec.ts'
+  ],
+  middleware: [
+    // Rewrites short-hand paths for test resources, making them fully relative to the repo root.
+    function rewriteResourcePath(context, next) {
+      if(context.url.startsWith('/resources/')) {
+        context.url = '/common/test' + context.url;
+      }
+
+      return next();
+    }
+  ],
+  plugins: [
+    esbuildPlugin({ts: true, target: 'auto'}),
+    importMapsPlugin({
+      inject: {
+        importMap: {
+          // Redirects `eventemitter3` imports to the bundled ESM library.  The standard import is an
+          // ESM wrapper around the CommonJS implementation, and WTR fails when it hits the CommonJS.
+          imports: {
+            'eventemitter3': '/node_modules/eventemitter3/dist/eventemitter3.esm.js'
+          }
+        }
+      }
+    })
```

Comment:
The new wrinkle as of this PR:  we use "import maps" to ensure the test engine picks up the ESM import version of `eventemitter3`, rather than the CJS version specified by its package.json.